daisy_integration_tests/step_create_snapshots.wf.json

```diff
@@ -0,0 +1,44 @@
+{
+  "Name": "create-snapshots-test",
+  "Vars": {
+    "about-this-test": {
+      "Value": "",
+      "Description": "This test creates an snapshot and confirm it's OK by creating disk from it."
+    }
+  },
+  "Steps": {
+    "create-disks": {
+      "CreateDisks": [
+        {
+          "name": "disk-from-image-family-url",
+          "sourceImage": "projects/debian-cloud/global/images/family/debian-9",
+          "type": "pd-ssd"
+        }
+      ]
+    },
+    "disks-to-snapshots": {
+      "CreateSnapshots": [
+        {
+          "name": "disks-to-snapshots",
+          "sourceDisk": "disk-from-image-family-url"
+        }
+      ]
+    },
+    "snapshots-to-disks": {
+      "CreateDisks": [
+        {
+          "name": "snapshots-to-disks",
+          "sourceSnapshot": "disks-to-snapshots"
+        }
+      ]
```

Comment:
Maybe include the disk size parameter as well? (in this test or another test)

Reply:
it's a feature of the google API and we don't do anything special for this parameter except for validation. I don't feel it's worth testing it in integration test. Let me cover it in unit test.